drivers/unix/os_unix.cpp

```diff
@@ -545,12 +545,6 @@ String OS_Unix::get_executable_path() const {
 		return OS::get_executable_path();
 	}
 	return b;
-#elif defined(__OpenBSD__) || defined(__NetBSD__)
-	char resolved_path[MAXPATHLEN];
-
-	realpath(OS::get_executable_path().utf8().get_data(), resolved_path);
```

Comment:
Seems like there's no API for it, so I guess we should check if `realpath(argv[0])` exist and if it's not also check in the `PATH`?

Reply:
For the reference: https://github.com/ziglang/zig/issues/6718